### DISCUSSION TITLE
Problemas de verificacion con railway

### DISCUSSION THREAD

**Discussion by fabianq846** [community/community#59055] [New to GitHub]
No me deja utilizar repositorios de github en railway, por problemas de verificaion en github. Dice que tiene problemas con verificaion de seguridad, necesito su ayuda, gracias

**Comment by queenofcorgis** [comment]
Hi @fabianq846 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

**Comment by fabianq846** [comment]
It won't let me use github repositories on Railway, due to github verification issues.  It says that it has problems with the security verification, I need your help, thanks
